Assets/Runtime/Window/WindowUILayer.cs

```diff
@@ -110,6 +110,11 @@ public class WindowUILayer : AUILayer<IWindowController>
             base.ReparentScreen(controller, screenTransform);
         }
 
+        public bool IsWindowOpened(string screenId)
+        {
+            return registeredScreens.TryGetValue(screenId, out var ctl) && CurrentWindow == ctl;
```

Comment:
window can be opened and not be current one

Reply:
as I know uiframework is designed to have only one active window at same time
@DevFrommars do I miss something?